stage one tests failing due to postgres options in Ubuntu

As per our maintainers call earlier this week, I am working towards to having a CI pipeline for our downstream releases.

The stage one tests upstream (here) doesn't trigger the postgresql plugin, so unit tests pass without any problems. But using GH runners it does get enabled, and we get issues as shown in the action link below

https://github.com/canonical/sos/actions/runs/24241138015/job/70776284710

The failure is due to alignment of the postgresql pghome variable, which is longer for the UbuntuPlugin class compared to RedHat one.

```
FAIL 1-tests/report_tests/help_output_tests.py:ReportListPluginsTest.test_plugin_formatting -> AssertionError: Regex didn't match: ' ([\\S ]){40} ([\\S ]{15}) ([\\S ])*' not found in ' postgresql.pghome                        /var/lib/postgresql psql server home directory'
```

So, we need to resolve this, so that this doesn't impact all of our plugins.

Something for future, we ought to enable the plugin that the PR is amending so that we catch the issues in advance, and not just run the default stage one tests that is then triggered by default. Maybe check for changed files in sos/report/plugins, and then append `-e <plugin>` for each plugin

This is what it looks like when you enable postgresql on an Ubuntu machine, and do -`-list-plugins`

```
 postgresql.pghome                        /var/lib/postgresql psql server home directory
 postgresql.username                      postgres        username for pg_dump
 postgresql.password                                      password for pg_dump (password visible in process listings)
 postgresql.dbname                                        database name to dump with pg_dump
 postgresql.dbhost                                        database hostname/IP address (no unix sockets)
 postgresql.dbport                        5432            database server listening port
```

--- a/sos/report/__init__.py
+++ b/sos/report/__init__.py
@@ -1099,7 +1099,7 @@ class SoSReport(SoSComponent):
                     tmpopt = 0
 
                 self.ui_log.info(f" {f'{opt.plugin}.{opt.name}':<40} "
-                                 f"{tmpopt:<15} {opt.desc}")
+                                 f"{tmpopt:<22} {opt.desc}")
         else:
             self.ui_log.info(_("No plugin options available."))
 

--- a/sos/report/plugins/juju.py
+++ b/sos/report/plugins/juju.py
@@ -65,15 +65,15 @@ class Juju(Plugin, UbuntuPlugin):
             "controllers",
             default="",
             val_type=str,
-            desc="Collect Juju state for specified controllers. Uses a \
-            whitespace delimited list.",
+            desc=("Collect Juju state for specified controllers. Uses a "
+                  "whitespace delimited list."),
         ),
         PluginOpt(
             "models",
             default="",
             val_type=str,
-            desc="Collect Juju state for specified models. Uses a whitespace \
-            delimited list.",
+            desc=("Collect Juju state for specified models. Uses a whitespace "
+                  "delimited list."),
         ),
     ]
 

--- a/tests/report_tests/help_output_tests.py
+++ b/tests/report_tests/help_output_tests.py
@@ -79,7 +79,7 @@ class ReportListPluginsTest(StageOneOutputTest):
         for opt in plugin_options:
             if not opt.strip():
                 continue
-            self.assertRegex(opt, r' ([\S ]){40} ([\S ]{15}) ([\S ])*')
+            self.assertRegex(opt, r' ([\S ]){40} ([\S ]{22}) ([\S ])*')
 
 
 class ReportListPresetsTest(StageOneOutputTest):

